chainntnfs/txnotifier.go

```diff
@@ -74,6 +74,15 @@ var (
 	// out of range.
 	ErrNumConfsOutOfRange = fmt.Errorf("number of confirmations must be "+
 		"between %d and %d", 1, MaxNumConfs)
+
+	// ErrEmptyWitnessStack is returned when a spending transaction has an
+	// empty witness stack.
+	//
+	// NOTE: due to unknown reasons bitcoind can return empty witness. See
+	// more in,
+	// - https://github.com/lightningnetwork/lnd/issues/5977
+	// - https://github.com/lightningnetwork/lnd/issues/7803
```

Comment:
This issue can be removed (we found the underlying problem). Tho one could add this one, because we cannot guarantee users don't set it, or have it in their old config files: https://github.com/bitcoin/bitcoin/issues/28730

Reply:
cool updated.